### DISCUSSION TITLE
Anyone able to share their generated SDXL engine?

### DISCUSSION THREAD

**Discussion by birep** [NVIDIA/Stable-Diffusion-WebUI-TensorRT#85] [General]
@contentis said elsewhere that the engines should be compatible within architectures. I have a 3060 with 12gb and can't generate the default engine for SDXL -- I run out of VRAM no matter what I try.

I'd be interested in trying to run someone else's generated engine, if anyone can share. It should work as long as it was generated on a 30xx/40xx GPU.

**Comment by contentis** [comment]
I won't be posting any compiled engines, but another user has done so: #47 https://www.youtube.com/watch?v=04XbtyKHmaE
